src/ui/Quickview/QuickviewDocument.ts

```diff
@@ -200,6 +200,9 @@ export class QuickviewDocument extends Component {
   private writeToIFrame(iframe: HTMLIFrameElement, content: HTMLDocument);
   private writeToIFrame(iframe: HTMLIFrameElement, content: String);
   private writeToIFrame(iframe: HTMLIFrameElement, content: any) {
+    _.each($$(content.body).findAll('a'), (link) => {
```

Comment:
I would add a sanity check here, because the method does accept content as either type (string or HTMLDocument), and while in practice it is always an HTMLDocument, we need to handle the fact that it could be a string and that code would throw null ref